src/reports/controllers/reportsController.js

```diff
@@ -87,3 +89,40 @@ module.exports.postStats = async (req, res, next) => {
     }
     return res.status(204).json();
 };
+
+module.exports.getExportedReport = async(req, res, next) => {
+    let exportedReport;
+    try{
```

Comment:
no need to try and catch inside try and catch
one try-catch for both operation is ok

Reply:
Thank you! I have adjusted the code accordingly.